CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Add `ensure(Int, NumRange)`
 * Add `skip(Iter, Int)`
 * Add `last(Eachable1, Any)`, the counterpart of first()
+* Add `skip(i:Iter, pattern=Bool.constructors)`
```

Comment:
`i:` - inconsistent with the surrounding lines. Why?

Reply:
should it be skip(Iter, Bool)?